NEWS.md

```diff
@@ -355,6 +355,8 @@
 `repo: jira_projects: [...]` and specifying a custom issue URL with
 `repo: url: issue: ...` in `_pkgdown.yml` (@jonkeane, #1466).
 
+* Tab title in tabsets may include HTML (@gadenbuie, #1915).
```

Comment:
I don't think this needs a bullet because tabs are only supported in the dev version?

Reply:
Makes sense, I removed the item from the news